### PR TITLE
LPS-143086 Can not export content page translation with experiences

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ExportTranslationMVCResourceCommand.java [hunk 1]
```diff
@@ -164,6 +154,16 @@ private void _addZipEntry(
...
 
+		InfoItemFieldValuesProvider<Object> infoItemFieldValuesProvider =
+			_infoItemServiceTracker.getFirstInfoItemService(
+				InfoItemFieldValuesProvider.class, className);
+
+		InfoItemObjectProvider<Object> infoItemObjectProvider =
+			_infoItemServiceTracker.getFirstInfoItemService(
+				InfoItemObjectProvider.class, className);
+
+		Object object = infoItemObjectProvider.getInfoItem(classPK);
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/ExportTranslationMVCResourceCommand.java:165]: As I say on https://github.com/liferay-lima/liferay-portal/pull/2532 the class created on that PR (InfoItemObjectProviderUtil) can be used to obtain the object and to reuse the code

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**